WcaOnRails/db/structure.sql

```diff
@@ -401,6 +402,22 @@ CREATE TABLE `Scrambles` (
 /*!40101 SET character_set_client = @saved_cs_client */;
 
 --
+-- Table structure for table `ar_internal_metadata`
+--
+
+DROP TABLE IF EXISTS `ar_internal_metadata`;
```

Comment:
Ah, I have no idea what the implications of keeping or removing this are. All I know is that it is something that was added in Rails 5. If we're going to add it, I'll admit that this PR is probably not the right place to do it, so I'll remove it =)